impeller/renderer/backend/gles/reactor_gles.cc

```diff
@@ -165,7 +190,13 @@ bool ReactorGLES::ReactOnce() {
   //----------------------------------------------------------------------------
   /// Collect all the handles for whom there is a GL handle sibling.
   ///
-  for (const auto& handle_to_collect : gl_handles_to_collect_) {
+  decltype(gl_handles_to_collect_) gl_handles_to_collect;
+  {
+    Lock ops_lock(ops_mutex_);
+    std::swap(gl_handles_to_collect_, gl_handles_to_collect);
+    FML_DCHECK(gl_handles_to_collect_.empty());
```

Comment:
Is this runtime asserting the behavior of swap?

Reply:
Umm, yeah. Thats possibly a bit much. I'll get rid of it in a subsequent patch.